src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -17438,11 +17520,16 @@ static void G()
         F(v).ToString();
     }
 }";
+            // Note: WRN_NullReferenceReceiver is reported for F(v).ToString() because
```

Comment:
PROTOTYPE?

Reply:
This is an existing issue. Logged #25778.

---
In reply to: [177568253](https://github.com/dotnet/roslyn/pull/25733#discussion_r177568253) [](ancestors = 177568253)